charts/timesketch/README.md

```diff
@@ -0,0 +1,286 @@
+<!--- app-name: Timesketch -->
+# Timesketch Helm Chart
+
+Timesketch is an open-source tool for collaborative forensic timeline analysis. 
+
+[Overview of Timesketch](http://www.timesketch.org)
+
+[Chart Source Code](https://github.com/google/osdfir-infrastructure)
+## TL;DR
+
+```console
+helm install timesketch-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/timesketch
+```
+> **Tip**: To quickly get started with a local cluster, see [minikube install docs](https://minikube.sigs.k8s.io/docs/start/).
+
+## Introduction
+
+This chart bootstraps a [Timesketch](https://github.com/google/timesketch/blob/master/docker/release/build/Dockerfile-latest) 
+deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.2.0+
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart, specify any release name of your choice. For example, if you
+want to install the chart for development, you can choose a release name of `timesketch-dev` then run:
+```console
+helm install timesketch-dev oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/timesketch
+```
+The command deploys Timesketch on the Kubernetes cluster in the default configuration. The [Parameters](#parameters) section lists the parameters that can be configured 
+during installation or see [Installating for Production](#installing-for-production) 
+for a recommended production installation.
+
+> **Tip**:  You can override the default Timesketch configuration by pulling the Helm
+chart locally and adding a `configs/` directory at the root of the Helm chart with user-provided configs.
+
+## Installing for Production
+
+Pull the chart locally and review the `values-production.yaml` file for a list of values that will be used for production.
```

Comment:
Should we specify the full path to this file?

Reply:
Per chat, it would be local to wherever the chart was pulled so if timesketch chart was pulled then it'd have its own `values-production.yaml` and osdfir chart will have its own `values-production.yaml`